Transição de tela Admin e Empregado

gerorsch opened the pull request:
Transições para as seguintes telas:

- cadastro de empregado;
- catálogo de bens;
- analise da proposta;
- devedor em detalhe.

Adiciona controladores para as telas principais do Admin e Empregado e atualiza as referidas telas.

gerorsch committed:
Adiciona controlador para Tela Principal do Admin

Adiciona as funções btnCadastrarEmpregadoPressed e btnCatalogoPressed

gerorsch committed:
fix: modifica a tela principal do Admin

gerorsch committed:
add: transição para a tela de bens e de cadastro de empregados

gerorsch committed:
Adiciona controlador para tela principal do empregado

Inclui as funções btnConfirmarDevedorPressed e btnAnalisePropostaPressed

gerorsch committed:
fix: atualiza a tela principal do empregado

gerorsch committed:
add: transição para as telas devedor em detalhe e analise de proposta

SteffanoP commented:
@gerorsch acho que só vou poder dar uma olhada nisso amanhã

SteffanoP reviewed (changes requested):
Parabéns @gerorsch! Sua implementação está perfeita! É exatamente assim que eu estava pensando as telas. Só gostaria que você pudesse trabalhar em só mais algumas coisinhas:

- Se possível, adicionar botões de voltar, por exemplo voltar para Login, voltar para tela principal do Admin.
- retirar o arquivo `src/out/production/projetoIP2/gui/TelaPrincipalEmpregadoController.class`, tal como o diretório de produção, visto que não faz parte do projeto.

De resto, estou muito feliz com seu desempenho @gerorsch!

gerorsch committed:
fix: arquivo removido

gerorsch committed:
feat: botão voltar

Adiciona à Tela de Análise de Proposta botão para voltar à tela principal

gerorsch committed:
feat: botão voltar

Adiciona à tela de catálogo de BENS o botão voltar à tela principal

gerorsch committed:
feat: botão voltar

Adiciona à tela Bens Empresa botão para voltar à tela principal do Admin

gerorsch committed:
feat: botão voltar

Adiciona à tela Cadastro Empregados botão para voltar à tela principal do Admin

gerorsch committed:
feat: botão voltar

Adiciona à tela Devedor em Detalhe botão voltar à tela principal do Empregado

gerorsch committed:
feat: botão voltar

Adiciona à tela Informações Pessoais botão voltar à tela principal do Empregado

gerorsch committed:
feat: botão voltar

Adiciona às telas principais do Admin e do Empregado botão para retornar à tela de Login

gerorsch committed:
fix: adiciona transição para a tela de catálogo de bens da empresa

gerorsch committed:
feat: método para botão retornar

Adiciona método à TelaAnalisePropostaController para retrornar à tela principal do Empregado

gerorsch committed:
feat: método para botão voltar

Adiciona à TelaBENSController método para retornar à tela principal do empregado

gerorsch committed:
feat: método para botão retornar

Adiciona à TelaBensEmpresaController método para retornar à tela principal do Admin

gerorsch committed:
feat: método para botão voltar

Adiciona à TelaCadastroEmpregadoController método para retornar à tela principal do Admin

gerorsch committed:
feat: método botão voltar

Adiciona à TelaDevedorDetalheController método para retornar à tela principal do empregado

gerorsch committed:
feat: método para botão voltar

Adiciona à TelaInformacoesPessoaisController método para retornar à tela principal do empregado

gerorsch committed:
feat: método para botão voltar

Adiciona à TelaPrincipalAdminController método para retornar à tela de Login.

Corrige a tela chamada pelo método btnCatalogoPressed.

gerorsch committed:
feat: método para o botão voltar

Adiciona à TelaPrincipalEmpregadoController método para retornar à tela de Login.

SteffanoP committed:
fix: botão voltar inconsistente com versão atual de `dev-gui`

SteffanoP committed:
remove: TelaBensEmpresaController.class

SteffanoP reviewed (approved):
Fiz pequenas correções de versão, tudo parece em ordem; Parabéns @gerorsch!

SteffanoP committed:
Merge branch 'dev-gui' into dev-gui-transicao-admin

SteffanoP committed:
fix: import não necessário
